CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
 # Change Log
 
+## [1.0.14]
```

Comment:
💡 If I may recommend: (Also we can chat up in internal or community call) - we should treat `Release` as separate email and do changelog as part of that exclusive reasons.

I hope I am able to explain this better but can detail more if not. (Essentially - treat normal PRs as Pull request to fix things, then `changelog` pr specific for `release` exclusivity.

Reply:
Removed